app/views/products/index.html.erb

```diff
@@ -1,51 +1,12 @@
-<% if notice %>
-  <aside id="notice"><%= notice %></aside>
+<% content_for :main_heading_action_button do %>
+  <%= render "shared/button", text: "New Product", link: new_product_path, style_class: 'Polaris-Button--primary' %>
 <% end %>
 
-<h1>Products</h1>
+<%= render partial: "shared/page_title", locals: { title: "Products" } %>
 
-<table>
-  <%# <thead>
-    <tr>
-      <th>Title</th>
-      <th>Description</th>
-      <th>Image url</th>
-      <th>Price</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead> %>
-
-  <tfoot>
-    <tr>
-      <td colspan="3">
-        <%= link_to 'New Product', new_product_path %>
-      </td>
-    </tr>
-  </tfoot>
-
-  <tbody>
-    <% @products.each do |product| %>
-      <tr class="<%= cycle('list_line_odd', 'list_line_even') %>">
-        <td class="image">
-          <%= image_tag(product.image_url, class: 'list_image') %>
-        </td>
-        
-        <td class="description">
-          <h1><%= product.title %></h1>
-          <p><%= truncate(strip_tags(product.description), length: 80) %></p>
-        </td>
+<% cache @products do %>
+  <%= render partial: "shared/product", collection: @products, locals: { manage: true } %>
```

Comment:
Will do, I think I did try it and hit an issue but will give it another go...

Reply:
Ah yes, I did try and the reason I can't do that is cos the `_product` partial is in the **shared** folder, not in the **products** folder - the shorthand needs it to be in the same folder and I reuse the `_product` partial in different views :)

Reply:
Ah great. I think further down in the review I realized this might've been the case but when I went looking for this comment I couldn't find it to verify that it was a shared partial.